### PR TITLE
wallet page "Bridge" button rename to -> "Bridge to alt L1"

### LINKED ISSUE #704: Gateway Wallet: There is no Alt L1 Bridge available as before.
Can you please share more details, what exactly we have to achieve here?

### PR DESCRIPTION
**Pull request by sk-enya**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

Describe what your Pull Request is about in a few sentences.

## Changes

Describe your changes and implementation choices. More details make PRs easier to review.

- Update name for bridge buttons to 'Bridge to L1'

## Testing

Screenshot 2023-04-27 at 11 45 56 AM

### CODE REVIEW

**Comment by InoMurko** [comment]
Do we need to continue using ALT?

**Comment by sk-enya** [comment]
> Do we need to continue using ALT?

Hi @InoMurko ,
Add this changes based on the  @Gian-Bobarino comment on https://github.com/bobanetwork/boba/issues/704#issuecomment-1511750494

**Comment by GianBobarino** [comment]
It can be as "Alt L1 Bridge" or "Bridge to Alt L1". But the issue is that having two "bridges" will confuse users. Better to be clear and concise.

**Comment by InoMurko** [comment]
> It can be as "Alt L1 Bridge" or "Bridge to Alt L1". But the issue is that having two "bridges" will confuse users. Better to be clear and concise.

But do we need to continue using the word ALT? It's just L1 no? Eth, Avax, Bnb... they're L1s. Cc @Boyuan-Chen ? Wdyt

**Comment by sk-enya** [comment]
@Gian-Bobarino changes will be available in next release cycle as it's too late and not blocker.

File: packages/boba/gateway/src/components/listToken/listToken.js [hunk 1]
```diff
@@ -186,7 +186,7 @@ function ListToken({ token, chain, networkLayer, disabled, loading,
                       tooltip="A multi-chain bridge to Alt L1."
                       fullWidth
                     >
-                      Bridge
+                      Bridge to alt L1
```
Comment (InoMurko) [inline packages/boba/gateway/src/components/listToken/listToken.js]: I would drop the alt here
```suggestion
                      Bridge to L1
```

File: packages/boba/gateway/src/components/listToken/listToken.js [hunk 2]
```diff
@@ -504,7 +504,7 @@ function ListToken({ token, chain, networkLayer, disabled, loading,
                   variant="contained"
                   fullWidth
                 >
-                  Bridge
+                  Bridge to alt L1
```
Comment (InoMurko) [inline packages/boba/gateway/src/components/listToken/listToken.js]:
```suggestion
                  Bridge to L1
```

**Review by InoMurko — APPROVED** [review]

**Review by alvaro-ricotta — APPROVED** [review]

**Pull request merged**